flink-ml-lib/src/main/java/org/apache/flink/ml/evaluation/binaryclassification/BinaryClassificationEvaluator.java

```diff
@@ -385,21 +257,19 @@ public void mapPartition(
 
             List<BinarySummary> statistics =
                     getRuntimeContext().getBroadcastVariable(partitionSummariesKey);
-            long[] countValues =
+            double[] countValues =
```

Comment:
This double variable 'countValues' is not a count yet. The name here is not suitable. Maybe 'weightSum' or 'accValue' is better.